app/lib/geordi-logger.coffee

```diff
@@ -4,12 +4,21 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
   constructor: (@state, @geordi) ->
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
+  @GEORDI_WARNINGS_TIMEOUT_INTERVAL = 5 * 60 * 1000 # only warn about Geordi down once every 5 mins, to avoid console spam
```

Comment:
Re the caps: I am using the convention of capitals for non-changing, constant values (see http://www.w3schools.com/js/js_conventions.asp) and mixed case when there is no guarantee the data won't change. (`tokens` could legitimately be changed at runtime).